ykrt/src/mt.rs

```diff
@@ -277,8 +277,10 @@ impl MT {
                         mtt.promotions.take().unwrap(),
                     )
                 });
+                self.stats.timing_state(TimingState::TraceMapping);
                 match thrdtrcr.stop() {
                     Ok(utrace) => {
+                        self.stats.timing_state(TimingState::None);
```

Comment:
why don't we time just stop and use an intermediate let? This way we don't need multiple "turn off timing" calls.